lib/handlers/user/index.js

```diff
@@ -65,6 +66,19 @@ handlers[EMsg.ClientUpdateMachineAuth] = function(data, callback) {
     }, new schema.CMsgClientUpdateMachineAuthResponse(response).toBuffer());
   });
 };
+handlers[EMsg.ClientNewLoginKey] = function(data) {
+  if(!this._logOnDetails.should_remember_password){
+    return;
```

Comment:
The behavior of message handlers shouldn't depend on some field in a previously sent message. This can lead to surprising behavior that will be difficult to document.